docs/paths/replies.yml

```diff
@@ -0,0 +1,98 @@
+PATH_replies:
+  post:
+    summary: Create new reply
+    description: Used to create a new reply in collection.
+    replyBody:
+      required: true
+      content:
+        application/json:
+          schema:
+            $ref: "../schemas/_index.yml#/createReplyPayload"
+    responses:
+      '201':
+        description: 'Created'
+        content:
+          application/json:
+            schema:
+              type: object
+              properties:
+                message: 
+                  type: string
+            example:
+              type: object
+              properties:
+                message: "Reply has successfully been created."
+
+  get:
```

Comment:
i think it might make more sense to add something like `GET /replies/{requestId}` which returns all of the replies associated with a given request